### PR TITLE
remove Common.Logging.Log4Net1211

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

File: exercises/shared-api-gateway/ITOps.EndpointConfig/EndpointConfigurationExtensions.cs [hunk 1]
```diff
@@ -18,7 +16,7 @@ public static class EndpointConfigurationExtensions
             string connectionString,
             Action<RoutingSettings<MsmqTransport>> configureRouting)
         {
-            NServiceBus.Logging.LogManager.Use<DefaultFactory>();
+            LogManager.Use<DefaultFactory>();
```
Comment (DavidBoike) [inline exercises/shared-api-gateway/ITOps.EndpointConfig/EndpointConfigurationExtensions.cs]: Is this even necessary?

Comment (adamralph) [reply exercises/shared-api-gateway/ITOps.EndpointConfig/EndpointConfigurationExtensions.cs]: I've no idea.

@Particular/logging-maintainers any idea what this is for? Completely redundant?

I guess I'll just try removing it and if I still get logs, all good!

**Comment by mauroservienti** [comment]
What’s the purpose of the `EndpointConfigurationExtensions` class? And why is it in the `shared-api-gateway` folder, that contains only API Gateway related code?

**Comment by adamralph** [comment]
@mauroservienti `EndpointConfigurationExtensions` applies all the endpoint config that common to all endpoints.

`shared-api-gateway/` is indeed mis-named, since the `ITOps.EndpointConfig` project was added to it. I'll rename it to `shared` in another PR. 😉

**Comment by mauroservienti** [comment]
Well, it’s the other way round since it was meant to contain API Gateway code only,  it I’m fine with making it more general purpose by renaming it to `shared` only

File: exercises/shared-api-gateway/ITOps.EndpointConfig/EndpointConfigurationExtensions.cs (hunk 1 above)
Comment (adamralph) [reply exercises/shared-api-gateway/ITOps.EndpointConfig/EndpointConfigurationExtensions.cs]: Removed, and logs still appeared.

**Comment by adamralph** [comment]
Commit amended and WIP removed.

**Pull request merged**